web-client/src/models/requests/index.ts

```diff
@@ -24,7 +24,13 @@ export enum RequestStatus {
   removed = 'removed',
 }
 
```

Comment:
Added this for clean code.  I thought of having a single type for RefType, but it is better that each collection have different types for later flexibility.